packages/react/src/progress-indicator/ProgressIndicator.test.tsx

```diff
@@ -75,4 +103,188 @@ describe('ProgressIndicator', () => {
 			expect(await axe(container)).toHaveNoViolations();
 		});
 	});
+	describe('With custom active item', () => {
+		function createList(
+			activeStatus: ProgressIndicatorItemStatus,
+			extraProperties: Partial<ProgressIndicatorItem>
+		): ProgressIndicatorProps {
+			return {
+				items: [
+					{
+						...extraProperties,
+						label: 'Done',
+						status: 'done',
+						isActive: activeStatus === 'done',
```

Comment:
All of these could just be `isActive: true`

Reply:
I don't think that's a good idea. It's not a real use case. This way of doing it checks that each of the real use cases behave as expected.

Reply:
I updated the unit tests to cover checking that isActive applies on the ones that it is supposed to and thros an error on the ones that it should not be applied to.